templates/plugin/abrt.yml.erb

```diff
@@ -0,0 +1,23 @@
+---
+:enabled: <%= scope.lookupvar("foreman_proxy::plugin::abrt::enabled") %>
+# Log file for the forwarding script.
+:abrt_send_log_file: <%= scope.lookupvar("foreman_proxy::plugin::abrt::abrt_send_log_file") %>
+# Directory where uReports are stored before they are sent
+:spooldir: <%= scope.lookupvar("foreman_proxy::plugin::abrt::spooldir") %>
+# Merge duplicate reports before sending (requires the satyr gem)
+:aggregate_reports: <%= scope.lookupvar("foreman_proxy::plugin::abrt::aggregate_reports") %>
+# Period (in seconds) after which collected reports are forwarded. Meaningful
+# only if smart-proxy-abrt-send is run as a daemon (not from cron).
+:send_period: <%= scope.lookupvar("foreman_proxy::plugin::abrt::send_period") %>
+<% if scope.lookupvar("foreman_proxy::plugin::abrt::faf_server_url") != :undef -%>
```

Comment:
I think the check is good, but as this template is in the same class (scope) as the parameters, you can just do `if @faf_server_url` I think, which works better over Puppet versions than checking for `:undef`.